runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerConfig.java

```diff
@@ -140,12 +140,20 @@ default int amount()
 		return -1;
 	}
 
+	@ConfigItem(
```

Comment:
Why was this moved?

Reply:
Just to reorganize each config item into getter/setter pairs, at some point another item was added between the getter/setter for that item. Of course it's not essential, but I was editing right next to it anyway.